egs/grid/VSR/conformer_ctc2/asr_datamodule.py

```
@@ -0,0 +1,435 @@
+    
+    def supervision_intervals(self, cuts: CutSet) -> Dict[str, torch.Tensor]:
+        
+        start_frames, nums_frames = zip(
+            *(
+                supervision_to_frames(
+                    sup, 0.04, cut.sampling_rate, max_frames=None
+                )
+                for cut in cuts
+                for sup in cut.supervisions
+            )
+        )
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)